pandas/io/formats/style.py

```diff
@@ -480,6 +481,10 @@ def to_latex(
             Set to ``True`` to structure LaTeX compatible with the {siunitx} package.
         encoding : str, default "utf-8"
             Character encoding setting.
+        convert_css : bool, default False
```

Comment:
do we need an option for this? why wouldn't we always do this?

Reply:
We wouldn't want to convert if different settings for html and latex are wanted or if more control over the latex formatting is needed. But in that case, a new styler instance could easily be used, right?

Reply:
right this is by-definition on `.to_latex` so i think its clear.

Reply:
Styler now allows styling to be coded in LaTeX **or** HTML-CSS format. In each separate case, which are completely flexible in each format, you don't want to convert.
But, if you have created an HTML-CSS Styler and it contains the limited attr-values that can now be converted to LaTeX then you should use the convert option. Shouldn't be set to default though.